flux_local/kustomize.py

```diff
@@ -182,10 +182,13 @@ async def _resource_lock(key: str) -> AsyncIterator[None]:
 class FluxBuild(Task):
     """A task that issues a flux build command."""
 
-    def __init__(self, ks: Kustomization, path: Path) -> None:
+    def __init__(
+        self, ks: Kustomization, path: Path, ignore_paths: list[str] | None = None
+    ) -> None:
         """Initialize Build."""
         self._ks = ks
         self._path = path
+        self._ignore_paths = list(ignore_paths or [])
```

Comment:
Given `ignore_paths` is already a list then the `list` prefix here is not needed